lib/perl/Genome/WorkflowBuilder/Command.pm

```diff
@@ -247,6 +247,11 @@ sub operation_type_attributes {
             $attributes{$expected_attributes{$name}} = $value;
         }
     }
+
+    unless (defined $attributes{'lsfQueue'}) {
+        $attributes{'lsfQueue'} = Genome::Config::get('lsf_queue_build_worker');
+    }
```

Comment:
I believe this key needs to match the value on [line 215](https://github.com/genome/genome/pull/1487/files/155a71a73af43a4d3124e87e54d0a84249609fad#diff-bdb6fd10fa2c2b74b604776c9d33f79dR215).